pages/performances/[id].js

```diff
@@ -0,0 +1,360 @@
+        {/* Event title */}
+        <h3>{props.children}</h3>
+
+        <div className={styles.judge__tab_stats}>
+          {/* Artistic Score */}
+          <span>ARTISTIC: ##</span>
+
+          {/* Technical Score */}
+          <span>TECHNICAL: ##</span>
+
+          {/* Cumulative Score */}
+          <span>CUMULATIVE: ##</span>
+        </div>
+
+        <div className={styles.judge__tab_awards}>
+          <h4>AWARD NOMINATIONS: </h4>
+        </div>
+      </button>
+    </div>
+  );
+}
+
+const EmptyComponent = () => {
+  return (
+    <div className={styles.performance_details__no_content}>
+      <div>
+        <h2>No Feedback Yet</h2>
+        <h3>Check again later</h3>
+        <img src={DancerRedJump} />
+      </div>
+    </div>
+  );
+};
+
+const PerformanceSummary = () => {
+  return (
+    <div>
+      <div className={styles.modal__container}>
+        <div className={styles.modal}>
+          <div>
+            <h2>Entry ID</h2>
+            <span>{`1`}</span>
+          </div>
+          <div>
+            <h2>Dancer(s)</h2>
+            <span>
+              {`Madeline, Sophia, Grace, Mia, Evelyn, Molly, Alexa, Ellyana, Kailan, Melissa, Savannah,
+          Sela, Raeya, Ella`}
+            </span>
+          </div>
+          <div>
+            <h2>Choreographer(s)</h2>
+            <span>{`Nancy Vaillancourt & Robbie-Lynn Schreindler`}</span>
+          </div>
+          <div>
+            <h2>School</h2>
+            <span>{`Bluevale Collegiate Institute`}</span>
+          </div>
+          <div>
+            <h2>Competition Level</h2>
+            <span>{`Advance 1`}</span>
+          </div>
+          <div>
+            <h2>Style</h2>
+            <span>{`Jazz`}</span>
+          </div>
+          <div>
+            <h2>Size</h2>
+            <span>{`Medium Group`}</span>
+          </div>
+        </div>
+      </div>
+      <div className={styles.performance_details__awards_list}>
+        <h2>Awarded:</h2>
+        <ul>
+          <li>{`Outstanding Performance Advance 1 Medium Group`}</li>
+          <li>{`1st Placement Award`}</li>
+        </ul>
+      </div>
+      <div className={styles.performance_details__score_content}>
+        <div className={styles.performance_details__score_card}>
+          <div style={{ textAlign: 'center' }}>
+            <h1>{`88`}</h1>
+            <h2>{`Technical`}</h2>
+          </div>
+        </div>
+
+        <div className={styles.performance_details__score_card}>
+          <div style={{ textAlign: 'center' }}>
+            <h1>{`88`}</h1>
+            <h2>{`ARTISTIC`}</h2>
+          </div>
+        </div>
+
+        <div
+          className={`${styles.performance_details__score_card} ${styles.performance_details__score_card_cumulative}`}
+        >
+          <div style={{ textAlign: 'center' }}>
+            <h1>{`88`}</h1>
+            <h2>{`CUMULATIVE`}</h2>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const JudgeFeedback = () => {
```

Comment:
The Judge Feedback is set to static data for now - can be changed when we add hook up the API